test/ibmq/test_experiment.py

```diff
@@ -220,6 +220,8 @@ def test_upload_experiment(self):
         self.assertEqual(credentials.project, new_exp.project)  # pylint: disable=no-member
         self.assertTrue(new_exp.uuid)
         self.assertTrue(new_exp.creation_datetime)
+        self.assertEqual(ExperimentVisibility.PRIVATE, new_exp.visibility)
+        self.assertIsNotNone(new_exp.owner, 'Owner should be set')
```

Comment:
I don't know why this is failing eslint:

https://github.com/Qiskit/qiskit-ibmq-provider/pull/829/checks?check_run_id=1513158086#step:5:10

>test/ibmq/test_experiment.py:224:29: E1101: Instance of 'Experiment' has no 'owner' member (no-member)

If this were a problem wouldn't the assertion fail the tests?

Reply:
I guess whatever causes this is also a problem for hub/group/project above, maybe a bug in pylint? Anyway I'll just disable the check for this line like those others.